pages/DSNP/UserData.md

```diff
@@ -0,0 +1,84 @@
+# User Data
+
+DSNP stores user-related data of well defined formats associated with a user's DSNP Identifier.
+Data may be public or stored in encrypted form.
+The [Avro](https://avro.apache.org) schema specification is used to define the binary representation of relevant data types.
```

Comment:
We should expand some here as to what user data is. Here's a shot at it, but it is just a start:

```suggestion
DSNP stores user-related data of well defined formats associated with a user's DSNP Identifier.
User-related data is for data about the user that is intended to be accessible across the network, but does not need to be widely announced and instead can be read just-in-time for a particular user.
Data may be public or stored in encrypted form.
The [Avro](https://avro.apache.org) schema specification is used to define the binary representation of relevant data types.
```